client.go

```diff
@@ -521,6 +521,10 @@ type SubscriptionOptions struct {
 
 	// Partition sets the stream partition to consume.
 	Partition int32
+
+	// ReadReplica sets client's ability to subscribe from an ISR replica
+	// instead of subscribing to leader
```

Comment:
See comment [here](https://github.com/liftbridge-io/liftbridge-api/pull/22/files#r404422469) about the semantics of this.

Reply:
Change has been made to read from a random ISR replica